cpu/stm32/include/vendor/Makefile

```diff
@@ -16,6 +16,7 @@ PKG_VERSION_l1=4a2f3cd6ea4bbc8e36daff0c7574b0fc1dfb7df7  # v2.3.0
 PKG_VERSION_l4=e442c72651e8d4757f6562acc14da949644944ce  # v1.6.1
 PKG_VERSION_l5=d922865fc0326a102c26211c44b8e42f52c1e53d  # v1.0.3
 PKG_VERSION_wb=44490652c88a513ec9a892c328db0246cf113987  # v1.4.0
+PKG_VERSION_wl=35631ff4d3a3cc361241af5f62190d0ea43bd65b  # v1.0.0
```

Comment:
Maybe this is why it isn't building?

Reply:
The build has a dependency on this [PR](https://github.com/RIOT-OS/RIOT/pull/16237). Can you please try now ? I have added a commit (force pushed) so that you can compile the same.

Reply:
No luck, I tried with this PR and with the dependency. Maybe @fjmolinas can confirm that I am just doing something wrong? (ie, checkout and make all to see if it builds).

Reply:
The `cpu/stm32/vectors/STM32WL55xx.c` and `cpu/stm32/include/irqs/wl/irqs.h` were probably (wrongly) generated with your first try but you didn't remove them in between. So even with #16237, they are not regenerated and you are still using the old ones.

BTW, I don't think #16237 is needed: it should directly be part of this PR. The changes there are only related to stm32wl and that doesn't need its own PR. If @akshaim (or any other reviewer) disagrees, that's fine, but at least rebase this PR on top of it to avoid the confusion that @MrKevinWeiss is experiencing when testing.

Reply:
@MrKevinWeiss  Can you please try `make -B clean all BOARD=nucleo-wl55jc` ?

Reply:
@aabadie I made it a separate PR since the change affects STM32 CPU's at large :)

Reply:
> I made it a separate PR since the change affects STM32 CPU's at large :)

So how do you expect it to be tested, on all supported CPUs ? Or just a green Murdock is enough ?
If it's the second case, then there's no need for a second PR just for a 3 line changes, that is quite orthogonal (since you took my version) to the current version.

Reply:
Okay. If you find it to be fitting to the current PR context I shall close #16237.